test/functional/page_objects/visual_builder_page.js

```diff
@@ -69,23 +81,81 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }) {
       }
       await input.pressKeys(browser.keys.NULL); // Release modifier keys
       await input.pressKeys(browser.keys.BACKSPACE); // Delete all content
-      await input.type(markdown);
-      await PageObjects.visualize.waitForRenderingCount(prevRenderingCount + 1);
     }
 
     async getMarkdownText() {
       const el = await find.byCssSelector('.tvbEditorVisualization');
+      await PageObjects.common.sleep(300);
```

Comment:
Is there a way to replace it with any `wait condition`?

Reply:
no, since getting element so fast and best option is use sleep with a timeout. Actual element text is 
![image](https://user-images.githubusercontent.com/8816260/52961335-bf18ab80-33ab-11e9-953f-490445b2d493.png) 

and current text for same element is 
![image](https://user-images.githubusercontent.com/8816260/52961364-d0fa4e80-33ab-11e9-9d0d-53357cc23666.png)